pkg/controller/postgresql.go

```diff
@@ -37,16 +37,22 @@ func (c *Controller) clusterListFunc(options api.ListOptions) (runtime.Object, e
 		return nil, fmt.Errorf("Can't extract list of postgresql objects: %s", err)
 	}
 
+	var activeClustersCnt int
 	for _, obj := range objList {
 		pg, ok := obj.(*spec.Postgresql)
 		if !ok {
 			return nil, fmt.Errorf("Can't cast object to postgresql")
 		}
+
+		if pg.Error != nil {
+			continue
+		}
 		c.queueClusterEvent(nil, pg, spec.EventSync)
 
 		c.logger.Debugf("Sync of the '%s' cluster has been queued", util.NameFromMeta(pg.Metadata))
+		activeClustersCnt++
 	}
-	if len(objList) > 0 {
+	if activeClustersCnt > 0 {
 		c.logger.Infof("There are %d clusters currently running", len(objList))
```

Comment:
Perhaps should the number of clusters that has errors. Would be useful when we have a background cleaner.